test/plugin/smartPay.nettest.ts

```diff
@@ -86,34 +85,27 @@ const fixtures: SmartPayTestFixture = [
   }
 ]
 
-for (const fixture of fixtures) {
-  let tools: EdgeCurrencyTools
+const fakeIo = makeFakeIo()
+const opts: EdgeCorePluginOptions = {
+  initOptions: { smartPayPublicAddress },
+  io: { ...fakeIo, fetch, random: size => new Uint8Array(32) },
+  log: fakeLog,
+  nativeIo: {},
+  pluginDisklet: fakeIo.disklet
+}
 
-  const WALLET_TYPE = fixture.WALLET_TYPE
+describe('SmartPay PIX parseUri', () => {
+  for (const fixture of fixtures) {
+    const WALLET_TYPE = fixture.WALLET_TYPE
 
-  const fakeIo = makeFakeIo()
-  const opts: EdgeCorePluginOptions = {
-    initOptions: { smartPayPublicAddress },
     // @ts-expect-error
-    // eslint-disable-next-line @typescript-eslint/no-unused-vars
-    io: { ...fakeIo, fetch, random: size => fixture.key },
-    log: fakeLog,
-    nativeIo: {},
-    pluginDisklet: fakeIo.disklet
-  }
-  // @ts-expect-error
-  const factory = edgeCorePlugins[fixture.pluginId]
-  const plugin: EdgeCurrencyPlugin = factory(opts)
+    const factory = edgeCorePlugins[fixture.pluginId]
```

Comment:
We can fix the type error by annotating ` pluginId: keyof typeof edgeCorePlugins` in your fixture type definition.

Reply:
good catch. done

Reply:
Nice catch. Done